src/gatsby/plugins/gatsby-plugin-openapi/gatsby-node.ts

```diff
@@ -6,6 +6,8 @@ import {
   RequestBody,
 } from "./types";
 
+import { RequestBodySchema } from "~src/gatsby/plugins/gatsby-plugin-openapi/types";
```

Comment:
@leeandher we should merge imports of types.

Reply:
whoops, will fix 👍  #justautoimportthings